gradle/dependencies.gradle

```diff
@@ -6,8 +6,9 @@ def versions = [:]
 
 versions.bytebuddy = '1.9.7'
 versions.junitJupiter = '5.1.1'
+versions.errorprone = '2.3.2'
 
-libraries.junit4 = 'junit:junit:4.12'
+libraries.junit4 = 'junit:junit:4.13-beta-1'
```

Comment:
I think the JUnit dependency should be kept at 4.12, we should avoid depending on a beta version.

Reply:
The checker API requires 4.13 sadly. However, I will revert it to just run 4.13 for the tests on this project. 4.13 will be released next week.

Reply:
OK waiting for junit 4.13 then

Reply:
I'm OK shipping with 4.13 beta because this new jar is completely decoupled from other jars. Hope that helps!